Assets/Scripts/Cards/CardDrawAnimation.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Utilities.Singletons;
+
+public class CardDrawAnimation : Singleton<CardDrawAnimation>
+{
+    // Start is called before the first frame update
+    private Vector3 p1DrawStartLocation;
+    private Vector3 p2DrawStartLocation;
+    public GameObject playerHandArea;
```

Comment:
I think you can call `PlayerHand.Instance.GetPlayerHandArea` or something - if it doesn't exist, I would make one. This way you don't need to have multiple components with the same UI element attached (can get messy when altering elements)